stories/molecules/Teaser.stories.tsx

```diff
@@ -16,48 +17,23 @@ type Story = StoryObj<typeof Teaser>;
 export const WithAllFields: Story = {
     name: 'With all fields',
     args: {
-        title: <span title={titleText}>{titleText}</span>,
-        subTitle: 'Awsome inc.',
-        location: 'Melbourne',
-        details: 'It was posted here',
-        statuses: [
-            {
-                label: 'Viewed',
-                tooltip: 'Viewed one day ago',
-            },
-            {
-                label: 'Imported',
-                tooltip: 'Imported two days ago',
-            },
-        ],
+        title: titleText,
+        subtitle: 'Awsome inc.',
+        hasCheckbox: true,
+        timestamp: 'Today',
+        primaryInfo: { text: 'Primary', href: 'https://textkernel.com' },
+        secondaryInfo: { text: 'Secondary' },
+        tercearyInfo: { text: 'Terceary' },
+        sourceInfo: { text: 'Source', icon: <FaLinkedinIn /> },
+        isVisited: true,
+        matchingIndicatorPercentage: 50,
+        tags: [<Tag bgColor="yellow">First</Tag>, <Tag bgColor="lightblue">Second</Tag>],
     },
-    render: (args) => <Teaser {...args} />,
-};
-
-export const WithOneStatus: Story = {
-    name: 'With one status',
-    args: {
-        title: <span title={titleText}>{titleText}</span>,
-        subTitle: 'Awsome inc.',
-        location: 'Melbourne',
-        details: 'It was posted here',
-        statuses: [
-            {
-                label: 'Viewed',
-                tooltip: 'Viewed one day ago',
-            },
-        ],
-    },
-    render: (args) => <Teaser {...args} />,
-};
+    render: (args) => {
+        const [isSelected, setIsSelected] = React.useState(false);
 
-export const WithoutStatus: Story = {
-    name: 'Without statuses',
-    args: {
-        title: <span title={titleText}>{titleText}</span>,
-        subTitle: 'Awsome inc.',
-        location: 'Melbourne',
-        details: 'It was posted here',
+        return (
+            <Teaser {...args} isSelected={isSelected} onChange={() => setIsSelected(!isSelected)} />
+        );
     },
```

Comment:
I think it would be good to add some example that show how it might look like for Search and for Jobs Data and duplicate some examples shown in Figma. Now it is pretty abstract for me when looking at storybook. 
It can be one story or several, but some more "real world" examples would be nice to see.

Reply:
Added